deps/copy-symlink-target.sh

```diff
@@ -4,7 +4,12 @@ set -eu
 
 main() {
   symlink="$1"
-  target="$(readlink -f "$symlink")"
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    # use greadlink from coreutils
+    target="$(greadlink -f "$symlink")"
+  else
+    target="$(readlink -f "$symlink")"
+  fi
```

Comment:
We're fine with this not being a widely used solution, right? Otherwise we might want to add something to at least the README.

Alternatively, maybe something like:

```suggestion
  if [[ "$OSTYPE" == "darwin"* ]]; then
    if command -v greadlink > /dev/null; then
      target="$(greadlink -f "$symlink")"
    else
      echo "Please install coreutils"
      exit 1
    fi
  else
    target="$(readlink -f "$symlink")"
  fi
```